src/main/java/org/gridsuite/network/map/model/ReactiveCapabilityCurveMapData.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.network.map.model;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+/**
+ * @author Seddik Yengui <seddik.yengui at rte-france.com>
+ */
+
+@Builder
+@Getter
+@EqualsAndHashCode
+public class ReactiveCapabilityCurveMapData {
+
+    private Double p;
```

Comment:
`@JsonInclude(JsonInclude.Include.NON_NULL)` ?

Reply:
When reactiveCapabilityCurvePoints of GneratorMapData is no null, the members of ReactiveCapabilityCurveMapData can not be null